src/controller.rs

```diff
@@ -498,24 +480,7 @@ fn pod_template(config: &WasmCloudHostConfig, _ctx: Arc<Context>) -> PodTemplate
         spec: Some(PodSpec {
             service_account: Some(config.name_any()),
             containers: containers.clone(),
-            volumes: Some(vec![
-                Volume {
-                    name: "nats-config".to_string(),
-                    config_map: Some(ConfigMapVolumeSource {
-                        name: Some(config.name_any()),
-                        ..Default::default()
-                    }),
-                    ..Default::default()
-                },
-                Volume {
-                    name: "nats-creds".to_string(),
-                    secret: Some(SecretVolumeSource {
-                        secret_name: Some(config.spec.secret_name.clone()),
-                        ..Default::default()
-                    }),
-                    ..Default::default()
-                },
-            ]),
+            volumes: Some(volumes.clone()),
```

Comment:
this `spec.Volumes` was getting overwritten in https://github.com/wasmCloud/wasmcloud-operator/blob/6d64c20b60a4821a4db7291e98b7f06882f06014/src/controller.rs#L494 
this change makes it consistent.